### ISSUE TITLE
about tabs adjust to  small resolution ratio(320px) the header scroll bar

### ISSUE THREAD

**Issue by UniqueCrownClown** [Dogfalo/materialize#2635]
About components Tabs,in 320px screen,the header scroll bar appear,feels it is not nice,how to full of the screen

**Comment by Frzy** [comment]
I've noticed this too.  Not just on 320px screen but as I grow or shrink the window I noticed a horizontal scrollbar flicker on the tab headers. See screenshot below.

**Browser**
![screen shot 2016-01-22 at 11 32 46 am](https://cloud.githubusercontent.com/assets/1053425/12519528/b09a6ece-c0fc-11e5-8e63-ef1efeb06705.png)

**Element**
![screen shot 2016-01-22 at 11 33 08 am](https://cloud.githubusercontent.com/assets/1053425/12519538/b824775c-c0fc-11e5-81ee-3702b6046341.png)

When Resizing the window a little scroll goes away

**Browser**
![screen shot 2016-01-22 at 11 33 27 am](https://cloud.githubusercontent.com/assets/1053425/12519550/cdf93fea-c0fc-11e5-9004-b3a927df1bd1.png)

**Element**
![screen shot 2016-01-22 at 11 33 35 am](https://cloud.githubusercontent.com/assets/1053425/12519554/d06d5554-c0fc-11e5-85a0-134c1c376d0d.png)


2 work-arounds.

To never show scroll bar on tabs add the following in _tabs.scss from
```css
.tabs {
    ...
    overflow: hidden;
}
```

or update the following lines in the tabs.js file

```js
// Line 41
$indicator.css({"right": $tabs_width - (($index + 1) * $tab_width)});
// Line 51
$indicator.css({"right": $tabs_width - (($index + 1) * $tab_width)});
// Line 95
$indicator.velocity({"right": $tabs_width - (($index + 1) * $tab_width)}, { duration: 300, queue: false, easing: 'easeOutQuad'});
// Line 101
$indicator.velocity({"right": $tabs_width - (($index + 1) * $tab_width)}, { duration: 300, queue: false, easing: 'easeOutQuad'});
```

to

```js
// Line 41
$indicator.css({"right": Math.max(0, $tabs_width - (($index + 1) * $tab_width))});
// Line 51
$indicator.css({"right": Math.max(0, $tabs_width - (($index + 1) * $tab_width))});
// Line 95
$indicator.velocity({"right": Math.max(0, $tabs_width - (($index + 1) * $tab_width))}, { duration: 300, queue: false, easing: 'easeOutQuad'});
// Line 101
$indicator.velocity({"right": Math.max(0, $tabs_width - (($index + 1) * $tab_width))}, {duration: 300, queue: false, easing: 'easeOutQuad', delay: 90});
```

This always will confine the indicator within the container never having a negative value

**Comment by acburst** [comment]
Fixed in 4f18709

**Comment by DavidUnzue** [comment]
Still having this issue with v0.97.8 ...
